www_src/main.less

```diff
@@ -94,3 +94,5 @@ body {
     width: 50px;
   }
 }
+
+@import "positionable";
```

Comment:
Yeah, that'd be good to figure out. I just nested my stuff inside of `#map-view` so it should play nice now!